src/website/src/app/documentation/demos/stepper/design-stepper.demo.html

```diff
@@ -0,0 +1,86 @@
+<p>
+  Each panel representing each step will contain a title and a description that describes the entire workflow at a
+  glance. After completing each step, collapsed panels then replace step descriptions with summaries describing what the
+  user has accomplished.
+</p>
+
+<p>
+  Here is when you might want to use a stepper over normal forms or wizards:
```

Comment:
This is a very important point of the documentation, IMO. Maybe we could add some highlight on this paragraph with the list below?
I also think we should have similar section in the wizard doc page. It's quite common for users to try to hack the wizard for inline use.

Reply:
Good point, I updated it to be a heading to stand out more.